src/eth-module.ts

```diff
@@ -0,0 +1,7 @@
+declare global {
```

Comment:
I believe we need this file to have the `.d.ts` extension for the declaration to register

Reply:
Updated